docs/usage/custom-controls/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Custom controls
+order: 5
+---
+
+# Custom controls
+
+Instead of using the [built-in control components](../controls.md) you can also integrate any existing components. The built-in components give you the benefit that everything is wired up for you automatically though. So when using custom components, you will need to wire some things up explicitly, pertaining data flow and accessible markup.
```

Comment:
A suggestion, no need to adjust though!

```suggestion
Instead of using the [built-in control components](../controls.md) you can also integrate any existing components in your codebase. The components provided by ember-headless-form give you the benefit that everything is wired up for you automatically. When using custom components, you will need to wire up data flow and accessibility.
```